website/docs/docs/build/cumulative-metrics.md

```diff
@@ -176,29 +186,33 @@ For example, let's consider an underlying measure of `order_total.`
         agg: sum
 ```
 
-We can compare the difference between a 1-month window and a monthly grain to date. The cumulative metric in a window approach applies a sliding window of 1 month, whereas the grain to date by month resets at the beginning of each month.
+We can compare the difference between a 1-month window and a monthly grain to date. 
+- The cumulative metric in a window approach applies a sliding window of 1 month
+- The grain to date by month resets at the beginning of each month.
 
 ```yaml
 metrics:
-  - name: cumulative_order_total_l1m  #For this metric, we use a window of 1 month 
+  - name: cumulative_order_total_l1m  # For this metric, we use a window of 1 month 
     label: Cumulative Order total (L1M)
     description: Trailing 1-month cumulative order amount
     type: cumulative
     type_params:
       measure: order_total
-      window: 1 month
-  - name: cumulative_order_total_mtd   #For this metric, we use a monthly grain-to-date 
+      window: 1 month # Applies a sliding window of 1 month
+  - name: cumulative_order_total_mtd   # For this metric, we use a monthly grain-to-date 
     label: Cumulative Order total (MTD)
     description: The month-to-date value of all orders
     type: cumulative
     type_params:
       measure: order_total
-      grain_to_date: month
+      grain_to_date: month # Resets at the beginning of each month
+      window_choice: min # Value at the beginning of the window
```

Comment:
@Jstein77 added window_choice here. What is the context for period_agg? shoudl i remove here?